common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -280,7 +280,7 @@ public String decrypt(final String encryptedBlob)
      * @throws GeneralSecurityException
      * @throws IOException
      */
-    synchronized SecretKey loadSecretKeyForEncryption() throws IOException,
+    synchronized public SecretKey loadSecretKeyForEncryption() throws IOException,
```

Comment:
nitpick: From the [JLS spec](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.4.3), it's customary [though not required] to declare access modifiers in the following order

```
public / protected / private 
abstract 
static 
final 
transient 
volatile 
synchronized 
native 
strictfp
```

For consistency, let's call it `public synchronized SecretKey loadSecretKeyForEncryption()` and apply the same naming to `loadSecretKeyForEncryption(String)`

Reply:
Fixed.